tests/runners/test_runners.py

```diff
@@ -220,17 +220,17 @@ def test_generate(self):
     for i, res in enumerate(stub.GenerateModelOutputs(req)):
       self._validate_response(res, text + out.format(i=i))
 
-  def test_stream(self):
-    text = "This is a long text for testing stream"
-    out = "Stream Hello World {i}"
-    req = self._format_request(text)
+  # def test_stream(self):
```

Comment:
Rather than commenting out, can we add skip with a TODO comment like [this one](https://github.com/Clarifai/clarifai-python/blob/2aaf180badafba9dc06218688de53bffad378ec8/tests/compute_orchestration/test_crud_operations.py#L107)?

Reply:
oh this is nice, let me update it